boards/arm/nrf51_pca10028/doc/nrf51_pca10028.rst

```diff
@@ -45,21 +45,33 @@ hardware features:
 +-----------+------------+----------------------+
 | Interface | Controller | Driver/Component     |
 +===========+============+======================+
-| NVIC      | on-chip    | nested vectored      |
-|           |            | interrupt controller |
 +-----------+------------+----------------------+
-| RTC       | on-chip    | system clock         |
-+-----------+------------+----------------------+
-| UART      | on-chip    | serial port          |
+| ADC       | on-chip    | adc                  |
 +-----------+------------+----------------------+
 | GPIO      | on-chip    | gpio                 |
 +-----------+------------+----------------------+
+| I2C       | on-chip    | i2c                  |
++-----------+------------+----------------------+
 | FLASH     | on-chip    | flash                |
```

Comment:
Isn't this list supposed to be ordered alphabetically? If so, FLASH should ge before GPIO, and SPI after RTT.

Reply:
thanks, will fix